content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -16,207 +16,255 @@ aliases:
-
-{{< note >}}
- You can optionally add any Custom Claims at this step. You must ensure they do not start with `jit_` or use any of the [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims). The Custom Claims can be used in an [Attribute-Based Access Control (ABAC) policy](/platform/administration/managing-access-to-an-asset-with-abac) to grant permissions. 
-{{< /note >}}
-
-{{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
-
-4.  You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint.
-
-{{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
-{{< /caution >}}
-
-{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
-
-5. Now that you have created your Custom Integration, follow the steps below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
-
 {{< note >}}
-**Note:** By default, newly created Integrations will always have a Non-Administrator permission to the API, you must add the Integration as an Administrator to elevate it's permissions.
-
-You can add a Custom Integration as an Administrator using the `Settings` screen, where the issuer will be `https://app.rkvst.io/appidpv1` and the subject will be your Integration's `CLIENT_ID`.
+**Note:** Creating App Registrations requires **Administrator** privileges.  
+If `Settings` does not appear in the navigation, see your RKVST Administrator for access.
 {{< /note >}}
 
-### Using the App Registrations API to create an Integration
+### Using the RKVST App to Create an Integration (First-Time Setup)
+
+1. As an Administrator, open the <a href="https://app.rkvst.io/" target="_blank">RKVST App</a>
+1. Navigate to `Settings` on the sidebar
+1. Navigate to the `Integrations` tab
+  {{< img src="IntegrationsTab.png" alt="Rectangle" caption="<em>Navigate to Settings, then Integration</em>" class="border-0" >}}
+1. Click the `Custom` box to create a Custom Integration
+1. Enter any `Display Name` you'd like
+  {{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
+  {{< note >}}
+  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+  {{< /note >}}  
+1. Once complete, click `Confirm` to complete the custom integration
+1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint
+    {{< caution >}}
```

Comment:
```suggestion
{{< caution >}}
```

Reply:
These spaces prevent the Caution box from displaying correctly.

Reply:
@pcnorth, curious why you're seeing this differently. Here's what I see locally:
![image](https://github.com/rkvst/rkvst-docs/assets/7647382/c34f2f7b-b145-43b1-aaf5-ea9c5b5e55a2)

What's also interesting: if I undent the block from lines 49-52, the ordered list still maintains the number counting. 🤔